release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -601,6 +601,11 @@ Kubernetes 1.27 removed the following deprecated API, so you must migrate manife
 
 |===
 
+[id="ocp-4-14-openshift-default-registry"]
+==== Removal of the OPENSHIFT_DEFAULT_REGISTRY 
+
+{product-title} {product-version} has removed support for the `OPENSHIFT_DEFAULT_REGISTRY` variable. This variable was primarily used to enable backwards compatibility of the internal image registry for legacy setups. 
```

Comment:
```suggestion
{product-title} {product-version} has removed support for the `OPENSHIFT_DEFAULT_REGISTRY` variable. This variable was primarily used to enable backwards compatibility of the internal image registry for earlier setups. 
```
From IBM Style Word Usage: "Do not use to refer to older software, applications (as in _legacy product_), values, programming languages, or technologies that are being supplemented or replaced. Avoid any implication that a product is old-fashioned or might not include up-to-date technologies. However, it is alright to refer to legacy data.

Because this usage can cause problems for our clients, specify the product information without any explanatory adjectives. For products that are still supported, but are not being actively developed, use _existing_, _traditional_, _established_, _classic_, _earlier_, or another appropriate neutral description in preference to legacy. To compare an existing value with a newer one, use words showing time relationships, such as earlier or previous."